Converter/CMakeLists.txt

```diff
@@ -28,14 +26,22 @@ set(${PROJECT_NAME}_SRCS
 
 set(${PROJECT_NAME}_HDRS
   igtlioBaseConverter.h
-  #igtlioVideoConverter.h
   igtlioImageConverter.h
   igtlioPolyDataConverter.h
   igtlioStatusConverter.h
   igtlioTransformConverter.h
   igtlioStringConverter.h
   )
 
+IF(${OpenIGTLink_ENABLE_VIDEOSTREAMING} EQUAL "1")
```

Comment:
I'm pretty sure this can be simplified to just `IF(OpenIGTLink_...)`

Reply:
en , you mean make OpenIGTLink_ENABLE_VIDEOSTREAMING an option?

Reply:
From [https://cmake.org/cmake/help/latest/command/if.html?highlight=#command:if](https://cmake.org/cmake/help/latest/command/if.html?highlight=#command:if)

> if(<constant>)
True if the constant is 1, ON, YES, TRUE, Y, or a non-zero number. False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND. Named boolean constants are case-insensitive. If the argument is not one of these specific constants, it is treated as a variable or string and the following signature is used.

Reply:
So if the value is 1, IF(var_name) will result in true

Reply:
thanks for the link !  :-)